documentation/user_docs/docs/manual/Correcting_for_sample_misalignment.rst

```diff
@@ -417,15 +417,14 @@ rlu_corr_to_lattice
 Extract lattice parameters and orientation matrix from r.l.u correction matrix and reference lattice parameters
 
 
-..
-   calc_proj_matrix
-   ================
+calc_proj_matrix
+================
```

Comment:
I don't know why this section was commented out in [PR#1224](https://github.com/pace-neutrons/Horace/pull/1224) - @oerc0122 do you remember?

Reply:
No idea. Though there are some outstanding comments in that PR which would be nice to have in this one.